src/main/java/com/hello2morrow/sonarplugin/api/SonargraphPlugin.java

```diff
@@ -48,13 +48,17 @@
   @Property(key = SonargraphPluginBase.REPORT_PATH, defaultValue = "", name = "Path of the Sonargraph report (empty means default value)", project = true, module = false,
     global = false)})
 public final class SonargraphPlugin extends SonarPlugin {
-
+  
   @SuppressWarnings("rawtypes")
   @Override
   public List getExtensions() {
     final List<Class<?>> list = new ArrayList<>();
     list.add(SonargraphRulesRepository.class);
-
+    try {
+            System.out.println("error");
+    } catch (Exception ex) {
+            ex.printStackTrace();

```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.foods.devops.mnscorp.net/coding_rules#rule_key=squid%3AS1148)